test/Microsoft.AspNet.Mvc.FunctionalTests/XmlSerializerInputFormatterTests.cs

```diff
@@ -14,8 +14,8 @@ namespace Microsoft.AspNet.Mvc.FunctionalTests
 {
     public class XmlSerializerInputFormatterTests
     {
-        private readonly IServiceProvider _services = TestHelper.CreateServices("XmlSerializerWebSite");
-        private readonly Action<IApplicationBuilder> _app = new XmlSerializerWebSite.Startup().Configure;
+        private readonly IServiceProvider _services = TestHelper.CreateServices(nameof(XmlFormattersWebSite));
```

Comment:
Our functional tests were spread out in different websites and was difficult to fit my scenarios in them and also I did not want to create a new one...instead I found that XmlSerializerWebsite was being used in only one test and felt it was easy to change it and to accommodate other Xml formatters too...